### ISSUE TITLE
Use alternate HttpClient?

### ISSUE THREAD

**Issue by MisterAcoustic** [luberda-molinet/FFImageLoading#1259]
I am trying to use my own instance of HttpClient with FFImageLoading, so that I can suppress the inclusion of any cookies in my image requests (to hopefully allow IIS to process the requests in parallel, rather than serially due to the presence of a session cookie).

### Steps to Reproduce

I use my own HttpClient to send requests from my Xamarin Forms app. I don't change the configuration of that instance, so it supports cookies through the built in CookieContainer, etc. Though my apps login process, a session cookie (our own) is established.

I am creating a second HttpClient instance configure to not use cookies, and setting that into the FFImageLoading default configuration as the HttpClient to be used:

`
public partial class App : Application
	{
        public System.Net.Http.HttpClient ImageLoadingClient = new System.Net.Http.HttpClient(
                                                                            new System.Net.Http.HttpClientHandler() { UseCookies = false, CookieContainer = null }
                                                                            );

        public App ()
		{

            InitializeComponent();

            ...

            FFImageLoading.Config.Configuration.Default.HttpClient = ImageLoadingClient;

            NavigationScreen start = new NavigationScreen();
            MainPage = new NavigationPage(start);
}
}
`
My images are loaded using a url with a CachedImage in a template for a Syncfusion ListView.

### Expected Behavior

I expect that my special HttpClient will be used, and no cookies should be included in the requests for my images.

### Actual Behavior

The session cookie established by my other HttpClient is always sent along with the image requests made from FFImageLoading.

### Basic Information

- Version with issue: 2.4.3.840
- Last known good version: N/A
- Platform: Xamarin Forms Android and iOS

**Comment by MisterAcoustic** [comment]
I do not currently believe this is an issue with FFImageLoading. I was finally able to separate out some calls to load images directly using my separate HttpClient, and I am getting the same behavior. In other words, even without FFImageLoading involved, I still see the cookie sent with the request. I must be missing something, but whatever it is, it isn't related to this library. Sorry if I wasted anyone's time.

If you happen to see this, and know what the issue really is, please go ahead and respond :).

Thanks.

**Comment by MisterAcoustic** [comment]
I just wanted to mention that while I was unable to figure out what was happening with my HttpClients, I was able to work around this issue by moving the controller function that serves the images to an entirely separate controller, which I marked with this attribute: [SessionState(SessionStateBehavior.Disabled)]

This is fine for our situation, and allows the server to process the requests in parallel. I'm sure the cookies are still being sent, but configured this way, the server is able to ignore them.

**Comment by daniel-luberda** [comment]
Hi @MisterAcoustic 

Did you try this?

```C#
var handler = new HttpClientHandler()
            {
                UseCookies = false,
            };
```

**Comment by MisterAcoustic** [comment]
Hi Daniel!

Yes - the code formatting of my original post is messed up, but that's exactly what I did first. In the code I posted, I also set the CookieContainer to null, but I did not do that originally.

I think the issue has something to do with how our server sets the session cookie - I couldn't get anything to change the behavior from the client side. My controller work-around has done the trick though. I no longer need to change the HttpClient that FFImageLoading is using - the built-in one is working fine with the modified controller.

Thanks!
